io.openems.edge.consolinno.pwm/readme.adoc

```diff
@@ -0,0 +1,7 @@
+= Consolinno Pwm
```

Comment:
Verstehe ich richtig, dass der `io.openems.edge.consolinno.*`-Namespace für alle Bundles gedacht ist, die zum Leaflet gehören? Wäre `io.openems.edge.consolinno.leaflet.*` o.Ä. sinnvoll um zukünftige Hardware-Versionen zu unterstützen? Ich spreche da aus Erfahrung bei FENECON....

Dieses Modul ist dann ein Leaflet-Modul für PWM, das via Modbus angesprochen wird?

Wenn sich ohnehin alle Komponenten auf die eine Leaflet-Hardware beziehen, könnten wir auch alle in einem einzigen Bundle unterbringen. Siehe z. B. die Implementierungen für GoodWe, in der Netzzähler, Charger, ESS, usw. alle in einem Bundle sind.

Reply:
Danke für den Vorschlag. Wir können die Bezeichnungen gerne anpassen.

Reply:
Ich würde wirklich vorschlagen, dass ihr alle Komponenten, die spezifisch für das Consolinno-Leaflet sind, in ein einziges Bundle zusammenführt. Ich sehe darin erstmal keinen Nachteil und es hält das Gesamtprojekt etwas übersichtlicher. Was hältst du davon?